app/src/main/java/info/nightscout/androidaps/plugins/general/autotune/AutotunePrep.kt

```diff
@@ -507,15 +532,17 @@ class AutotunePrep @Inject constructor(
         }
 
 // categorize.js Lines 437-444
+        //aapsLogger.debug(LTag.AUTOTUNE, "CRData: ${crData.size} CSFGlucoseData: ${csfGlucoseData.size} ISFGlucoseData: ${isfGlucoseData.size} BasalGlucoseData: ${basalGlucoseData.size}")
         if (verbose)
-            log("CRData: " + crData.size + " CSFGlucoseData: " + csfGlucoseData.size + " ISFGlucoseData: " + isfGlucoseData.size + " BasalGlucoseData: " + basalGlucoseData.size)
+            log("CRData: ${crData.size} CSFGlucoseData: ${csfGlucoseData.size} ISFGlucoseData: ${isfGlucoseData.size} BasalGlucoseData: ${basalGlucoseData.size}")
```

Comment:
why is not used aapsLogger?

Reply:
Because it is exported in a dedicated log file for autotune run (in autotune sub folder).
This dedicated log file has the same informations than openAPS log file and I can directly compare both results (categorization, profile tuning on basal, isf, ic and additional log informations included in openAPS).
I also export all files used by autotune with openAPS structure: entries, treatments, intermediate calculated profiles, and settings of autotune run (timeZone, command line to lauch on VM with start/end date, options...) to be able to run OpenAps with the same dataset and compare.
Before Unit Tests I only had this log file to compare and analyse results with openAPS run...